_inc/lib/class.media-summary.php

```diff
@@ -21,6 +24,11 @@ static function get( $post_id, $blog_id = 0, $args = array() ) {
 			$blog_id = get_current_blog_id();
 		}
 
+		$cache_key = md5( "{$blog_id}_{$post_id}_{$args['max_words']}_{$args['max_chars']}" );
```

Comment:
What's the point of using `md5()` here? Probably the md5 sum will be even longer than the value.